packages/cactus-plugin-ledger-connector-go-ethereum-socketio/src/test/typescript/unit-test/validatorDriver_getNonceHex.ts

```diff
@@ -60,33 +60,33 @@ import { io } from "socket.io-client";
     reqID: reqID,
   };
 
-  const json2str = (jsonObj) => {
+  const json2str = (jsonObj: object) => {
```

Comment:
@rwat17 Are these unrelated linter fixes? (which are great by the way) I'm fine with it either way but I do wnat to make sure it is clarified what this actually is in terms of purpose.

Reply:
TypeScript required to type these arguments while building at some point.